src/network.zig

```diff
@@ -21,6 +21,26 @@ const Vec3f = vec.Vec3f;
 const Vec3i = vec.Vec3i;
 const NeverFailingAllocator = main.utils.NeverFailingAllocator;
 
+pub const DisconnectType = enum(u8) {
```

Comment:
I don't particularly like that this is just declared randomly at the beginning of the file.
Wouldn't it make more sense to move it into the `disconnect` protocol?